src/main/scala/io/iohk/ethereum/blockchain/sync/fast/PivotBlockSelector.scala

```diff
@@ -185,11 +188,11 @@ class PivotBlockSelector(
       .map { case (_, bestPeerBestBlockNumber) => bestPeerBestBlockNumber }
```

Comment:
In the future maybe we can decouple the function “collectVoters” of this class, and reduce the complexity. By this way the function is moved to another class and we can add more unit tests. What do you think?

Reply:
I am not sure about that. For me, this function doesn't have much sense in other contexts. It is collecting voters for pivot block election and only for that